website_portal_sale/__openerp__.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'Website Portal for Sales',
+    'category': 'Website',
+    'summary': (
+        'Add your sales document in the frontend portal (sales order'
+        ', quotations, invoices)'
+    ),
+    'version': '8.0.1.0.0',
+    'author': 'Odoo SA, '
+              'MONK Software, '
+              'Antiun Ingenieria SA, '
```

Comment:
Please @sergio-incaser make a PR to FIX this
s/Antiun Ingenieria SA,/Antiun Ingeniería S.L. (review template)
Thanks